site/content/docs/quickstart/_index.md

```diff
@@ -0,0 +1,509 @@
+---
+linktitle: "Quickstart using Docker"
+weight: 1
+---
+
+<!-- TODO: Fix dates. Cross-platform date generation is a pain, so maybe use Docker locally? See what Netlify supports, or maybe there is a Hugo variable, or create a shortcode -->
+
+# Creating a Single Node M3DB Cluster with Docker
+
+This guide shows how to install and configure M3DB, create a single-node cluster, and read and write metrics to it.
+
+{{% notice warning %}}
+Deploying a single-node M3DB cluster is a great way to experiment with M3DB and get an idea of what it has to offer, but is not designed for production use. To run M3DB in clustered mode, with a separate M3Coordinator [read the clustered mode guide](/docs/how_to/cluster_hard_way).
```

Comment:
`in clustered mode, with a separate M3Coordinator` > `in clustered mode with a separate M3Coordinator,`